topiary/tests/samples/input/ocaml.ml

```diff
@@ -982,4 +982,8 @@ let _ =
 
 (* Slight variant of #441 *)
 let _ =
-  let module Foo = struct let f = 0 let g = 1 in ()
+  let module Foo = struct let f = 0 let g = 1 end in ()
+
+(* Indentation/newlines after non-specific infix operators *)
+let _ = foo bar @@ let () = qux in
+  baz
```

Comment:
Can we have an example of the form

```ocaml
myBlockIntroducingFunction @@ fun x ->
  something horrible onto x
```

as well? So that we can at least know for ourselves how these are formatted?

Reply:
Sure, I will add that. It should be formatted as
```ocaml
myBlockIntroducingFunction @@
  fun x ->
    something horrible onto x
```